mlflow/server/js/src/experiment-tracking/pages/experiment-scorers/SampleScorerOutputPanelContainer.tsx

```diff
@@ -26,7 +26,7 @@ const SampleScorerOutputPanelContainer: React.FC<SampleScorerOutputPanelContaine
   const scorerName = useWatch({ control, name: 'name' });
   const llmTemplate = useWatch({ control, name: 'llmTemplate' });
   const guidelines = useWatch({ control, name: 'guidelines' });
-  const scorerType = useWatch({ control, name: 'scorerType' });
+  const modelValue = useWatch({ control, name: 'model' });
```

Comment:
note: using `watch`ed value since RHF won't trigger "required" error after mount. Alternatively we can use `isValid` for entire form